client/components/Users.js

```diff
@@ -16,28 +16,20 @@ export class Users extends React.Component {
 
     }
 
-    componentDidMount(){
-        // I'll change the code below once we confirm the users data
-        let tempArrOfUsers = []; // Added 10 users from faker as dummy data
-        while(tempArrOfUsers.length < 11){
-            tempArrOfUsers.push(faker.name.firstName())
-        }
-        this.setState({users: tempArrOfUsers})
+    async componentDidMount(){
+        await this.props.getUsers()
+        this.setState({users: this.props.users})
     }
 
-
     async handleDeleteUser(ev) {
```

Comment:
I think this is never called

Reply:
I'm going to get rid of this and leave the delete function inside render.